proposed/0002-release-schedule.md

```diff
@@ -91,31 +95,16 @@ if those branches were less than two years old at the time.
 - Active users of tagged releases can upgrade as new releases are announced using the major/minor numbers as guidance as per semver standards.
 - Users wanting minimal upgrade hassle always have at least 2 years to get up to date to the latest major version for another two years or more.
 
-### Alternative 2
-
-We will follow the example of Ubuntu, which bases the Major Version Number on the Year.  
-And also has a specific Rythmus of an LTS version every 4 Releases,
-adding bigger experimantal features in the Major Release after an LTS version.
-
-But we will go with just one Major Release each year, which results in one LTS Release every 4 years.
-
-
-#### Visualization of the Ubuntu Release cycle
-
-![ubuntu release stragey visualized](https://github.com/OpenMage/rfcs/raw/main/assets/Ubuntu_release_cycle_Ubuntu.png?raw=true "ubuntu release strategy visualized")
-
 ## Implementation
 
-- Vote on the alternatives above and choose the winner
-- Designate the appropriate "default" branch in GitHub admin as per the winning strategy
-- Rename and/or delete inactive branches as necessary (perhaps after some time after an announcement in case they are being actively used)
-- Update all existing PRs to the appropriate branch (can be done gradually)
-- Lock old branches (1.9.4.x and 20.0) to avoid accidental merges if possible.
-- Codify the strategy into the main project `README.md` file
+- Create a "main" branch from "20.0" and designate it as the "default" branch in GitHub admin
+- Create a "next" branch that is a copy of "main"
+- Create a "v19" branch from "1.9.4.x"
+- Create a "v20" branch from "20.0"
+- Add branch protection rules for "main", "next", and "^v\d+$"
+- Add branch protection rule to lock old branches ("1.9.4.x" and "20.0") to avoid accidental merges
+- Delete branches "19.4.22" and "20.0.19"
```

Comment:
I think these are temporary branches created by @fballiano for the next release.

Reply:
they will be removed in a couple of days max

Reply:
```suggestion
```

those branches do not exist anymore